.github/pull_request_template.md

```diff
@@ -62,6 +62,7 @@ It is the responsibility of the author to make sure the pull request is ready to
 
 - [ ] [🛠][1] [Documentation](https://docs.esmvaltool.org/en/latest/community/dataset.html#dataset-documentation) is available
 - [ ] [🛠][1] The dataset has been [added to the CMOR check recipe](https://docs.esmvaltool.org/en/latest/community/dataset.html#testing)
+- [ ] [🛠][1] The dataset has been added to the shared [ESMValTool data pools](https://docs.esmvaltool.org/en/latest/community/dataset.html#cmorized-data)
```

Comment:
It might be helpful to add some more text here so that it is clear that the data will be moved by a member of the obs-maintainers team. Otherwise I can imagine people trying to move the data themselves to the OBS folders on Levante and/or JASMIN.

Reply:
That's a good point, @hb326. I have expanded the text a bit in 71e0edf. Would that be ok for you?

Reply:
@remi-kazeroni: yes, looks great! Thanks!